tests/test_transaction.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+from happy_bank_core.logic.transaction import Transaction, TransactionException
+from happy_bank_core.logic.account import Account
+
+customer_1 = Account("id123", "John Doe", 100)
+customer_2 = Account("id321", "Johanna Doe", 300)
```

Comment:
These test objects you can setup in setup method within TestTransaction class: https://docs.pytest.org/en/latest/how-to/xunit_setup.html#how-to-implement-xunit-style-set-up